public/templates/header.php

```diff
@@ -18,6 +18,7 @@
     <script src="//code.jquery.com/ui/1.12.1/jquery-ui.js"></script>
 
     <link rel="stylesheet" href="css/header.css">
+    <link rel="stylesheet" href="css/course_lecture.css">
     <link rel="stylesheet" href="css/email.css">
 
```

Comment:
Not sure how good of an idea this is , I assume that this CSS file is only used on specific pages?

Reply:
I did it like this because I saw chris do it this way lol, I'll change it I agree with you (I'll put it on the individual pages)

Reply:
done